### DISCUSSION TITLE
OneTable Demo with Onehouse, Microsoft, Google

### DISCUSSION THREAD

**Discussion by kywe665** [apache/incubator-xtable#249] [Show and tell]
Here is the recording for the first public demo of OneTable: https://opensourcedatasummit.com/open-data-foundations-with-onetable/
![onetable_demo](https://github.com/onetable-io/onetable/assets/1703248/f1a23587-d3ab-436c-bf91-74d450272a62)

Here is a copy of the slide deck: https://drive.google.com/file/d/1VXRdR0QgGn7JWs1-b3KV1MSBh9xHDXXi/view

Kudos to @the-other-tim-brown, @ashvina, and @anoopj for the presentation!

**Comment by sagarlakshmipathy** [comment]
The source code for the docker demo @the-other-tim-brown walked through is present here https://github.com/onetable-io/onetable/tree/main/demo
